src/Generation/MethodDetails.php

```diff
@@ -536,6 +536,11 @@ public function isStreaming(): bool
             $this->methodType === static::CLIENT_STREAMING;
     }
 
+    public function isServerStreaming(): bool
+    {
+        return $this->methodType === static::SERVER_STREAMING;
+    }
```

Comment:
I think this function may be unnecessary...

Reply:
I explained in the previous comment relating to this. We want to generate a rest descriptor config for server streaming, but _not_ the other two types of streaming (because we haven't been).